contrib/make_libsecp256k1.sh

```diff
@@ -20,15 +20,15 @@ set -e
 
 . $(dirname "$0")/build_tools_util.sh || (echo "Could not source build_tools_util.sh" && exit 1)
 
-here=$(dirname $(realpath "$0" 2> /dev/null || grealpath "$0"))
+here=$(dirname "$(realpath "$0" 2> /dev/null || grealpath "$0")")
```

Comment:
I guess you also need quotes for the top-level then.
```suggestion
here="$(dirname "$(realpath "$0" 2> /dev/null || grealpath "$0")")"
```
(and same goes in the other files)

Reply:
It wasn't needed to pass the `cd "$CONTRIB"` on line 31. But even with this, it gives an error near the end (I don't know where to change the line that don't have quotes `libtool: install: /usr/bin/install -c .libs/libsecp256k1.dll.a /home/tadeubas/test folder spaces/electrum/contrib/secp256k1/dist/lib/libsecp256k1.dll.a`:

```sh
make  all-am
make[1]: Entering directory '/home/tadeubas/test folder spaces/electrum/contrib/secp256k1'
  CC       src/libsecp256k1_la-secp256k1.lo
  CC       src/libsecp256k1_precomputed_la-precomputed_ecmult.lo
  CC       src/libsecp256k1_precomputed_la-precomputed_ecmult_gen.lo
  CCLD     libsecp256k1_precomputed.la
  CCLD     libsecp256k1.la
make[1]: Leaving directory '/home/tadeubas/test folder spaces/electrum/contrib/secp256k1'
make  install-am
make[1]: Entering directory '/home/tadeubas/test folder spaces/electrum/contrib/secp256k1'
make[2]: Entering directory '/home/tadeubas/test folder spaces/electrum/contrib/secp256k1'
 /usr/bin/mkdir -p '/home/tadeubas/test folder spaces/electrum/contrib/secp256k1/dist/lib'
 /bin/bash ./libtool   --mode=install /usr/bin/install -c   libsecp256k1.la '/home/tadeubas/test folder spaces/electrum/contrib/secp256k1/dist/lib'
libtool: install: /usr/bin/install -c .libs/libsecp256k1.dll.a /home/tadeubas/test folder spaces/electrum/contrib/secp256k1/dist/lib/libsecp256k1.dll.a
/usr/bin/install: target 'spaces/electrum/contrib/secp256k1/dist/lib/libsecp256k1.dll.a' is not a directory
make[2]: *** [Makefile:892: install-libLTLIBRARIES] Error 1
make[2]: Leaving directory '/home/tadeubas/test folder spaces/electrum/contrib/secp256k1'
make[1]: *** [Makefile:1650: install-am] Error 2
make[1]: Leaving directory '/home/tadeubas/test folder spaces/electrum/contrib/secp256k1'
make: *** [Makefile:1643: install] Error 2
🗯  ERROR: Could not install secp256k1
```

Reply:
ah, so the script still fails... hmm